server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandlerHelper.java

```diff
@@ -12,24 +12,80 @@
 import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.Struct;
+import java.util.Arrays;
+import java.util.List;
 
 /**
 * The StorageHandlerHelper class holds helper query functions that get the number of results
 * matching the primary keys of different Cloud Spanner tables.
 */
 public class StorageHandlerHelper {
+
+  /**
+  * Returns a Boolean that indicates whether or not a person is a member of a club.
+  * This method creates a Struct that holds a single row from a database read row transacation.
+  * It returns false if the Struct is null, indicating the membership does not exist.
+  * It returns true if the Struct is not null, indicating the membership does exist.
+  *
+  * @param  readContext  the context for an attempt to perform a transaction
+  * @param  userId       the user ID string of the user we are checking is in a club
+  * @param  clubId       the club ID string of the club we are checking the user is in
+  * @return Boolean      the Boolean true or false representing if the membership exists or not
+  */
+  public static Boolean checkMembership(ReadContext readContext, String userId, String clubId) {
+    Struct row =
+          readContext
+            .readRow(
+              "Memberships",
+              Key.of(userId, clubId),
+              Arrays.asList("userId"));
+    return (row != null) ? true : false;
+  }
+
+  /**
+  * Returns a long that is the number of members in a club, including the owner.
+  *
+  * @param  dbClient  the database client
+  * @param  userId    the user ID string used to query and get how many users have that key
+  * @return count     the long representing the number of results from the query
+  */
+  public static long getMemberCount(DatabaseClient dbClient, String clubId) {
+    long count = 0;
+    Statement statement = 
+        Statement.newBuilder(
+                "SELECT COUNT(*) as count "
+                  + "FROM Memberships "
+                  + "WHERE clubId = @clubId AND "
+                  + "(membershipType = " + MembershipConstants.MEMBER
+                  + " OR membershipType = " + MembershipConstants.OWNER + ")")
+              .bind("clubId")
+              .to(clubId)
+              .build();
+    try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
+      while (resultSet.next()) {
+        count = resultSet.getLong("count");
+      }
+    }
+    return count;
+  }
+
   /**
-  * Returns a long that is the number of results matching a specific user ID
+  * Returns a long that is the number of results matching a specific user ID.
+  *
   * @param  dbClient  the database client
   * @param  userId    the user ID string used to query and get how many users have that key
   * @return count     the long representing the number of results from the query
   */
-  public static long getPersonCountQuery(DatabaseClient dbClient, String userId) {
+  public static long getPersonCount(DatabaseClient dbClient, String userId) {
```

Comment:
Since this can only be 1 or 0, maybe return a boolean and rename to `doesPersonExist`

Reply:
Actually, this was a mistake on my part. `getPersonCount`, `getClubCount`, and `getBookCount` are no longer necessary functions that are called within StorageHandler.java since a readRow returns either a single row or null if it doesn't exists (which I am already checking for within the code). I will be removing these functions.